shared/middleware/helpers/redis_table.rb

```diff
@@ -0,0 +1,108 @@
+# RedisTable provides a table abstraction that notifies clients of changes.
+# Operations include adding a row, updating and fetching a specific row
+# by id, and fetching all rows. Rows preserve the order of insertion.
+# # The implementation wraps a RedisProperty bag to store the table rows
+# # and a pub sub service to notify clients of changes.
+
+require_relative 'redis_property_bag'
+
+# @private
+class RedisTable
```

Comment:
What do you mean by marking this class `@private`?  Isn't `RedisTable` the only symbol this file provides?